stepfunctions-lambda/Makefile

```diff
@@ -0,0 +1,69 @@
+export AWS_ACCESS_KEY_ID ?= test
+export AWS_SECRET_ACCESS_KEY ?= test
+export AWS_DEFAULT_REGION = us-east-1
```

Comment:
Do we intentionally overwrite the default region here or should it be `?=` ?

Reply:
On standard template, its `export AWS_DEFAULT_REGION=us-east-1`